composer.json

```diff
@@ -78,5 +79,11 @@
             "phpseclib/bootstrap.php",
             "phpseclib/Crypt/Random.php"
         ]
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "phpseclib4\\Tests\\": "tests/",
```

Comment:
For the 1.0 branch this line can be deleted. The tests in the 1.0 branch aren't namespaced and, if they were, they'd be namespaced with `phpseclib\` - not `phpseclib4\`.